app/src/main/java/com/imashnake/animite/features/searchbar/SearchBar.kt

```diff
@@ -51,7 +51,7 @@ import com.imashnake.animite.R as Res
 @OptIn(ExperimentalAnimationApi::class, ExperimentalMaterial3Api::class)
 @Composable
 fun SearchBar(
-    modifier: Modifier,
+    modifier: Modifier = Modifier,
```

Comment:
It pains me that the only non-optional param here is last, so it *requires*
`SearchBar(navController = ... )`
 instead of 
 `SearchBar(navController)`